packages/suite/src/components/suite/modals/CriticalCoinjoinPhase/PhaseProgress.tsx

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
 import styled from 'styled-components';
-import { CoinjoinSession, RoundPhase } from '@wallet-types/coinjoin';
+import { RoundPhase, SessionPhase } from '@trezor/coinjoin/src/enums';
+import { CoinjoinSession } from '@wallet-types/coinjoin';
```

Comment:
import SessionPhase from '@wallet-types/coinjoin' ? same as it previously was with RoundPhase

Reply:
done